readme.md

```diff
@@ -1 +1,3 @@
 Ilana Loves Justin Bieber
+
+Mike really love Justin Bieber and the above is farce.
```

Comment:
Is this grammatically correct? I think it should be "a farce."
![image](https://cloud.githubusercontent.com/assets/3011748/21621654/3ae73f50-d1b7-11e6-89ab-8f398d53ba81.png)